files/openhim/docker-compose.yml

```diff
@@ -0,0 +1,52 @@
+version: '3'
+ 
+services:
+  mongo:
+    container_name: mongo
+    image: mongo:3.4
+    ports:
+      - "27017:27017"
+    volumes:
+      - openhim-data:/data/db
+    restart: always
+    healthcheck:
+      test: echo 'db.stats().ok' | mongo localhost:27017/test --quiet
+      interval: 1m
+      timeout: 3s
+      retries: 3
+ 
+  openhim-core:
+    container_name: openhim-core
+    image: jembi/openhim-core
+    environment:
+      mongo_url: mongodb://mongo/openhim
+      mongo_atnaUrl: "mongodb://mongo/openhim"
+    ports:
+      - "8080:8080"
+      - "5000:5000"
+      - "5001:5001"
+    depends_on:
+      - mongo
+    restart: always
+    healthcheck:
+      test: ["CMD", "curl", "-fk", "https://localhost:8080/heartbeat"]
+      interval: 1m30s
+      timeout: 5s
+      retries: 3
+ 
+  openhim-console:
+    container_name: openhim-console
+    image: jembi/openhim-console
+    ports:
+      - "80:80"
```

Comment:
Port 80 is already being used on the host, but any 808* is free 💃

Reply:
I'll change this. Thanks.